validator/accounts/v2/wallet_create.go

```diff
@@ -148,14 +168,16 @@ func inputRemoteKeymanagerConfig(cliCtx *cli.Context) (*remote.Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &remote.Config{
+	newCfg := &remote.Config{
 		RemoteCertificate: &remote.CertificateConfig{
-			ClientCertPath: clientCrtPath,
-			ClientKeyPath:  clientKeyPath,
-			CACertPath:     caCrtPath,
+			ClientCertPath: strings.TrimRight(clientCrtPath, "\r\n"),
+			ClientKeyPath:  strings.TrimRight(clientKeyPath, "\r\n"),
+			CACertPath:     strings.TrimRight(caCrtPath, "\r\n"),
 		},
-		RemoteAddr: remoteAddr,
-	}, nil
+		RemoteAddr: strings.TrimRight(remoteAddr, "\r\n"),
+	}
+	fmt.Printf("%s\n", newCfg)
```

Comment:
Can this log be prettier?

Reply:
Oh, is this using the string function below? That's neat. Can you add it to the PR desc?

Reply:
Yep! it prints out nicely given newCfg implements the stringer interface